### PR TITLE
DDF-1277: reorganized DDF Spatial docs.

### PR DESCRIPTION
**Pull request by ricklarsen**
@stustison @clockard @djblue @vinamartin @gordocanchola @oscaritoro

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/460" target="_blank">ReviewNinja

**Comment by oscaritoro** [comment]
LGTM :+1:

File: catalog/spatial/docs/src/main/resources/ExtContents.adoc [hunk 1]
```diff
@@ -1,5 +1,27 @@
 
-== Overview
-The {branding} Spatial Application provides KML transformer and a KML network link endpoint that allows a user to generate a View-based KML Query Results Network Link.
+This section supports developers creating extensions of the existing framework.
+
+=== Whitelist
+
+The following packages have been exported by the DDF Spatial Application and are approved for use by third parties:
```
Comment (oscaritoro) [inline catalog/spatial/docs/src/main/resources/ExtContents.adoc]: DDF -> {branding> ?

**Comment by clockard** [comment]
LGTM :+1:

File: catalog/spatial/docs/src/main/resources/ExtContents.adoc (hunk 1 above)
Comment (ricklarsen) [inline catalog/spatial/docs/src/main/resources/ExtContents.adoc]: fixed.

**Comment by oscaritoro** [comment]
Ship it :+1:

**Comment by gordocanchola** [comment]
:+1: Built successfully and a tests passed.

**Pull request merged**